python-client/hatchet_sdk/clients/dispatcher.py

```diff
@@ -23,8 +24,10 @@ def get_action_listener(self, ctx, req):
     def send_action_event(self, ctx, in_):
         raise NotImplementedError
 
-DEFAULT_ACTION_LISTENER_RETRY_INTERVAL = 5  # seconds
+DEFAULT_ACTION_LISTENER_RETRY_INTERVAL = 1  # seconds
```

Comment:
should this be configurable in the yaml?

Reply:
Hmm, definitely eventually - going to leave it as-is for now though, since I'm not sure if we add exponential backoff before making this configurable.